src/app/js/formats/distributionChart/global-radar-chart/RadarChartAdmin.js

```diff
@@ -92,6 +169,15 @@ class RadarChartAdmin extends Component {
                     style={styles.input}
                     value={colors}
                 />
+                {this.createUI()}
+                <br />
```

Comment:
Can't you manage breaking spaces using CSS margins?
I know it can be complicated, though.

Reply:
Oh yeah I now see how it's made, I'll to go for that if needed :+1: